test/extensions/transport_sockets/tls/integration/ssl_integration_test.cc

```diff
@@ -352,6 +353,98 @@ TEST_P(SslCertficateIntegrationTest, ServerRsa) {
   checkStats();
 }
 
+// Server configured on SPIFFE certificate validation for mTLS
```

Comment:
The inclusion of SPIFFE tests in ssl_integration_test breaks builds for folks that want to include the TLS extension but not the SPIFFE cert verifier extension.  Would it be possible to move these tests to a separate test target?

Reply:
Thanks, I did in https://github.com/envoyproxy/envoy/pull/15324